package.json

```diff
@@ -1,83 +1,85 @@
 {
   "name": "pci-dss-sanitizer",
-  "version": "0.0.0-development",
+  "version": "0.0.0-development.0",
   "description": "Mask bank card info from strings",
-  "main": "lib/index.js",
-  "husky": {
-    "hooks": {
-      "pre-commit": "npm run lint",
-      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
-      "pre-push": "npm run check-coverage"
-    }
+  "license": "MIT",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "files": [
+    "dist",
+    "src"
+  ],
+  "engines": {
+    "node": "16"
   },
+  "peerDependencies": {},
   "scripts": {
-    "prebuild": "npm run lint && rimraf lib",
-    "lint": "eslint --cache src/**/*.js",
-    "format": "npm run lint -- --fix",
-    "build": "rollup -c",
-    "watch": "rollup -c --watch",
-    "test": "cross-env NODE_ENV=test jest",
+    "start": "tsdx watch",
+    "build": "tsdx build",
+    "test": "npm run test:client && npm run test:src",
+    "test:src": "tsdx test",
+    "test:client": "npm_scripts/test-client.sh",
     "tdd": "npm run test -- --watch",
+    "lint": "tsdx lint",
+    "format": "npm run lint -- --fix",
+    "prepare": "husky install && tsdx build",
+    "size": "size-limit",
```

Comment:
The `size-limit` dependency is not in the `package.json`

Reply:
And doesn't really need to be, I'll drop this command as well